smart_ats/jobs/tests/test_api.py

```diff
@@ -221,3 +220,124 @@ def test_list_with_DoesNotExist_company_id(self):
             "/api/v1/companies/-1/jobs/", content_type="application/json"
         )
         self.assertEqual(response.status_code, status.HTTP_404_NOT_FOUND)
+
+
+class JobApplicationTest(APITestCase):
+    def setUp(self):
+        self.job = JobFactory()
```

Comment:
```suggestion
        self.job = JobFactory(status='ACTIV')
```
self.job = JobFactory.create(status=Job.STATUSES.ACTIV)